mappings/net/minecraft/world/biome/layer/layerUtil/InitLayer.mapping

```diff
@@ -1,6 +1,7 @@
-CLASS chl net/minecraft/world/biome/layer/InitLayer
+CLASS chl net/minecraft/world/biome/layer/layerUtil/InitLayer
```

Comment:
Packages should be all lowercase (`layerutil`). But why not just make it `util`, since it's a subpackage of layer.